rllib/env/vector/vector_multi_agent_env.py

```diff
@@ -1,10 +1,10 @@
-import gymnasium as gym
-import numpy as np
+from typing import Any, Dict, List, Optional, Tuple, TypeVar
 
+import gymnasium as gym
 from gymnasium.core import RenderFrame
 from gymnasium.envs.registration import EnvSpec
 from gymnasium.utils import seeding
-from typing import Any, Dict, Optional, Tuple, TypeVar
+import numpy as np
```

Comment:
Always thought the style guide needs usual import first before relative imports?

Reply:
Oh, not sure, actually. It's always:
1) basic python imports (those you don't have to pip install)
2) library dependencies (e.g. gymnasium, numpy; those you DO have to pip install)
3) everything `ray`